src/app/core/learning-object-module/editor.service.ts

```diff
@@ -21,7 +21,7 @@ export class EditorService {
     authorUsername: string,
     status: string,
   ): Observable<string> {
-    const route = LEGACY_USER_ROUTES.UPDATE_MY_LEARNING_OBJECT(authorUsername, id);
+    const route = LEARNING_OBJECT_ROUTES.UPDATE_LEARNING_OBJECT_STATUS(id);
```

Comment:
Can this be moved to the learning object service?

Reply:
i removed the entire route because it wasn't used here, but we should probably remove the entirety of `EditorService` too because it's not referenced anywhere.

i'll keep `EditorService` for now though